pkgs/misc/drivers/argononed/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitLab, dtc, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "argononed";
+  version = "0.3.3";
+
+
+  src = fetchFromGitLab {
+    owner = "DarkElvenAngel";
+    repo = pname;
+    rev = "d50040115beece36a0259cef2248eff467286207";
+    sha256 = "sha256-3GBEzBh5VuuPh+iZeLBV/7oTX0Zqs8pcvfOXgDt68WQ=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  buildInputs = [ dtc ];
+
+  patchPhase = ''
+    patchShebangs configure
+  '';
+
+  installPhase = ''
+    install -Dm755 build/argononed $out/bin/argononed
+    install -Dm755 build/argonone-cli $out/bin/argonone-cli
+    install -Dm755 build/argonone-shutdown $out/lib/systemd/system-shutdown/argonone-shutdown
+    install -Dm644 build/argonone.dtbo $out/boot/overlays/argonone.dtbo
+
+    install -Dm644 OS/_common/argononed.service $out/lib/systemd/system/argononed.service
+    install -Dm644 OS/_common/argononed.logrotate $out/etc/logrotate.d/argononed
+    install -Dm644 LICENSE $out/share/argononed/LICENSE
+
+    installShellCompletion --bash --name argonone-cli OS/_common/argonone-cli-complete.bash
+  '';
+
+
```

Comment:
So does here. Have you pass it under nixpkgs-fmt?

Reply:
Sorry about the lines, saw a few packages formatted like that and thought that was the standard.

Reply:
![image](https://user-images.githubusercontent.com/5727578/161411956-d68c7532-7043-4cdf-bf81-e4185bc4cd63.png)
It seems nixpkgs-fmt is not catching those